src/libstubs.cpp

```diff
@@ -5,11 +5,13 @@
 // Library stubs -- function definition stubs when not using CSE definitions
 //                  e.g., for RCDEF and unit tests
 
+#if 0
 #include "cnglob.h"
 #include "vrpak.h"
 
 VROUTINFO5 PriRep = { { 0 } };
 const char* InputFilePathNoExt = NULL;
+#endif
```

Comment:
Do we need this or not? If it is sticking around, should have some comments about why.

Reply:
I agree it should be deleted, will do that.